core/src/main/scala/org/apache/spark/ui/SparkUI.scala

```diff
@@ -175,20 +178,22 @@ private[spark] object SparkUI {
       listener
     }
 
-    val environmentListener = new EnvironmentListener
     val storageStatusListener = new StorageStatusListener
-    val executorsListener = new ExecutorsListener(storageStatusListener)
     val storageListener = new StorageListener(storageStatusListener)
+    val environmentListener = new EnvironmentListener
+    val executorsListener = new ExecutorsListener(storageStatusListener)
+    val memoryListener = new MemoryListener
```

Comment:
nit: can you avoid reordering the lines you don't change?  Just for using git blame later on